funnel/views/commentvote.py

```diff
@@ -173,6 +177,69 @@ def new_comment(self):
         )
         return {'form': commentform_html}
 
+    @route('subscribe', methods=['POST'])
+    @requires_login
+    @render_with(json=True)
+    def subscribe(self):
+        csrf_form = forms.Form()
+        if csrf_form.validate_on_submit():
+            self.obj.add_subscriber(actor=current_auth.user, user=current_auth.user)
+            db.session.commit()
+            return {
+                'status': 'ok',
+                'message': _("You have successfully subscribed to this comment thread"),
+            }
+        else:
+            return {
+                'status': 'error',
+                'error_code': 'subscribe_error',
+                'error_description': _(
+                    "There was an issue subscribing to this comment thread. Please try again"
```

Comment:
This language won't do. The message must explain the problem. Also, no "please".